### PR TITLE
docs: copy editing

### PR DESCRIPTION
**Pull request by dkearns**

### CODE REVIEW

File: docs/autotools.rst [hunk 1]
```diff
@@ -19,17 +19,19 @@ Changing the executable's name
 
 On some systems, like certain BSDs, there is already a 'ctags' program in the base
 system, so it is somewhat inconvenient to have the same name for
-universal-ctags. During the ``configure`` stage you can now change
+Universal-ctags. During the ``configure`` stage you can now change
 the name of the created executable.
 
-To add a prefix 'ex' which will result in 'ctags' being transformed into 'exctags'
+To add a prefix 'ex' which will result in 'ctags' being renamed 'exctags':
```
Comment (codebrainz) [inline docs/autotools.rst]: "...being renamed to 'exctags'" also works.

File: docs/autotools.rst [hunk 2]
```diff
@@ -19,17 +19,19 @@ Changing the executable's name
...
 	$ ./configure --program-prefix=ex
 
-To completely change the program's name run the following. Please remember there is also an 'etags' installed alongside 'ctags'
+To completely change the program's name run the following:
 
 .. code-block:: bash
 
 	$ ./configure --program-transform-name='s/ctags/my_ctags/; s/etags/myemacs_tags/'
+
+Please remember there is also an 'etags' installed alongside 'ctags'.
```
Comment (codebrainz) [inline docs/autotools.rst]: "...alongside 'ctags' which you may also want to rename as shown above."

File: docs/windows.rst [hunk 3]
```diff
@@ -5,24 +5,24 @@ Building/hacking/using on MS-Windows
 
 ----
 
-This part of the documentation is written by Frank Fesevur, co-maintainer of universal-ctags and the maintainer of the Windows port of this project. It is still very much work in progress. Things still need to be written down, tested or even investigated. When building for Windows you need to know there are many compilers and many building environments. This is a summary of available options and things that have been tested so far.
+This part of the documentation is written by Frank Fesevur, co-maintainer of Universal-ctags and the maintainer of the Windows port of this project. It is still very much work in progress. Things still need to be written down, tested or even investigated. When building for Windows you need to know there are many compilers and many building environments. This is a summary of available options and things that have been tested so far.
```
Comment (codebrainz) [inline docs/windows.rst]: "...When building for Windows you should be aware that there are many compilers and build environments available."

File: docs/windows.rst [hunk 4]
```diff
@@ -5,24 +5,24 @@ Building/hacking/using on MS-Windows
...
 
 
 Compilers
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-There are many compilers for Windows. Apart from the ones mentioned here others do exist, but not used by me.
+There are many compilers for Windows. Apart from the ones mentioned here others do exist, but are not used by me.
```
Comment (codebrainz) [inline docs/windows.rst]: Last sentence could be "Compilers not mentioned here may work but are not tested."

File: docs/windows.rst [hunk 5]
```diff
@@ -5,24 +5,24 @@ Building/hacking/using on MS-Windows
...
 
 
 Microsoft Visual Studio
 .............................................................................
 http://www.visualstudio.com/
 
-Obviously there is Microsoft Visual Studio 2013. Many professional developers targeting Windows use Visual Studio. Visual Studio comes in a couple of different editions. Their Express Edition is free to use, but a Microsoft-account is required to download the .iso and when you want to continue using it after a 30-days trial period. All other editions you need to be paid for.
+Obviously there is Microsoft Visual Studio 2013. Many professional developers targeting Windows use Visual Studio. Visual Studio comes in a couple of different editions. Their Express Edition is free to use, but a Microsoft-account is required to download the .iso and when you want to continue using it after a 30-days trial period. All other editions need to be paid for.
```
Comment (codebrainz) [inline docs/windows.rst]: Newer "Express" editions are called "Community" editions now I think.

Last sentence could be "Other editions of Visual Studio must be purchased."

File: docs/windows.rst [hunk 6]
```diff
@@ -5,24 +5,24 @@ Building/hacking/using on MS-Windows
...
 
 Installing Visual Studio will give you the IDE, the command line compilers and the MS-version of make named nmake.
 
-Note that ctags can not be build with Visual Studio older than 2013 anymore. There is C99 (or C11) coding used that generate syntax errors with VS2012 and older. This could effect compilers from other vendors as well.
+Note that ctags can not be built with Visual Studio older than 2013 anymore. There is C99 (or C11) coding used that generates syntax errors with VS2012 and older. This could effect compilers from other vendors as well.
```
Comment (codebrainz) [inline docs/windows.rst]: `s/effect/affect/`

File: docs/windows.rst [hunk 7]
```diff
@@ -88,7 +88,7 @@ MSYS2 is a more maintained version of MSYS, but specially geared towards MinGW-w
 
 **Cygwin**
 
-Cygwin provides ports of many GNU/Linux tools and a POSIX API layer. This is the most complete way to get the GNU/Linux terminal feel under Windows. Cygwin has a setup that helps you install all the tools you need. Drawback of Cygwin is that the POSIX API layer (cygwin1.dll) is not fast.
+Cygwin provides ports of many GNU/Linux tools and a POSIX API layer. This is the most complete way to get the GNU/Linux terminal feel under Windows. Cygwin has a setup that helps you install all the tools you need. A drawback of Cygwin is that the POSIX API layer (cygwin1.dll) is not fast.
```
Comment (codebrainz) [inline docs/windows.rst]: Last sentence could read "One drawback of Cygwin is that it has poor performance."

File: docs/windows.rst [hunk 8]
```diff
@@ -104,7 +104,7 @@ You can also build a native Windows version using Autotools.
 
 If you use Autotools you can also do the Units testing with `make units`. Some tests fail, that needs to be investigated.
 
-Sometimes anti-virus product can slowdown the build and test process significantly, especially when ``./configure`` is running and during the Units tests. In that case it could help to temporary disable them. But be aware of the risks when you disable your anti-virus product.
+Sometimes anti-virus products can slowdown the build and test process significantly, especially when ``./configure`` is running and during the Units tests. In that case it could help to temporary disable them. But be aware of the risks when you disable your anti-virus product.
```
Comment (codebrainz) [inline docs/windows.rst]: Could begin "Some anti-virus software slows down the build...".

Last sentence could use "software" instead of "product".

**Comment by dkearns** [comment]
Thanks @codebrainz 

Any objections to those changes @ffes?

**Comment by ffes** [comment]
Changes LGTM. English is not my native language, and I tend to make this kind of mistakes. Great that they get fixed.

**Comment by masatake** [comment]
@codebrainz, thank you.

**Comment by masatake** [comment]
LGTM. Thank you all.

File: docs/windows.rst (hunk 5 above)
Comment (dkearns) [inline docs/windows.rst]: It looks like Community and Express are both still offered so I just mentioned Community as well.

**Pull request merged**